minimal.js

```diff
@@ -28,9 +44,26 @@ function toKeyValueTimestamp(msg) {
   }
 }
 
-module.exports = function (dirname, opts) {
+function isString (s) {
+  return 'string' === typeof s
+}
+
+module.exports = function (dirname, keys, opts) {
   var hmac_key = opts && opts.caps && opts.caps.sign
+
+  var codec = {
+    encode: function (obj) {
```

Comment:
I'm getting some:

could not retrive msg: NotFoundError: Key not found in database [@5yP0SoZVkMIXUQcL7nPGsw6yYIxE23UBd+rLO+6OSWc=.ed25519,5]

Shouldn't this be just JSON.stringify? I can understand overwriting decode.

Reply:
This seems to fix the createUserStream bug.